examples/README.md

```diff
@@ -1,32 +1,36 @@
 ## Examples
 
-In order for new users to get up and running with the packer builder, a few examples of building a machine image with popular distros have been created. At the time of this writing there are examples for the latest Ubuntu (20.04) and Centos (8.3) releases. Please open an issue if you'd like to see an example for another distro.
+In order for new users to get up and running with the packer builder, a few examples of building a machine image with popular distros have been created.
 
 In order to see an exhaustive list of configuration options for the packer builder please see the [following documentation](../docs/builders/xenserver-iso.html.markdown). This doc will focus on the details relevant to the particular distro.
 
 ### Running the examples
 
-In order to run this example you will need to perform the following steps:
-1. Export the `XAPI_HOST`, `XAPI_USERNAME` and `XAPI_PASSWORD` environment variables to the current shell. Note: The `XAPI_HOST` must be the resource pool primary.
-2. Run the `packer build` command specifying the storage repositories to use for the ISO upload and for the VM created during the build.
-
+In order to run the examples you will need to perform the following steps:
+1. Export those vars:
 ```
-# Replace sr_name and sr_iso_name with your storage repositories names
-packer build -debug  --var sr_name='Local storage' --var sr_iso_name=LocalISO examples/centos8.json
+PKR_VAR_remote_host
+PKR_VAR_remote_password
+PKR_VAR_remote_username
+PKR_VAR_sr_name
+PKR_VAR_sr_iso_name
+``` 
+`PKR_VAR_remote_host` must be the resource pool primary, aka the master.
 
-# Do the same variable replacement for the ubuntu example as well.
-packer build -debug  --var sr_name='Local storage' --var sr_iso_name=LocalISO examples/ubuntu-2004.json
-```
+2. Run `packer init path/to/defenition.pkr.hcl` to download the xenserver plugin
+
+2. Run `packer build  path/to/defenition.pkr.hcl`   
+so for example:
+`packer build  examples/centos/centos8-netinstall.pkr.hcl`
```

Comment:
Placeholder comment to remind me to run these commands to make sure they work from scratch.

Reply:
They all work.